Update docs.

mattwelke opened the pull request:
## Description
The first real update to the repo to add our docs contents.

* Removes Docusaurus and Facebook branding. Replaces it with a slight modification of the built in theme to replace the green color palette with a blue one. New graphics from [SVG Repo](https://www.svgrepo.com) and [undraw](https://undraw.co/) that use blue and orange as their color scheme. This is designed to match the blue and orange we got from our AI-generated logo.
* Replaces placeholder docs content with content for Validator. A Community page links them to all the ways they can get in touch with us.
* Removes `test-deploy.yml` GHA workflow. It was redundant because we have Netlify trying to build our site which is set up as a required check. If site builds are broken, PRs cannot be merged.

I also wanted to add a call to action to the splash page (maybe a "read more" button to bring them to the first docs page) but I couldn't figure out the CSS for now. Maybe we can add that later.

When this is merged, other changes should be made to remove documentation that we're centralizing here:

* https://github.com/validator-labs/validatorctl/pull/205
* Clear away description from GitHub org page, replace it with something short and sweet with a link to the new docs site.

Commit:
Update docs.

Remove Docusaurus and Facebook branding.

Replace placeholder docs content with content for Validator.

Remove test-deploy.yml GHA workflow. It was redundant because we have Netlify trying to build our site which is set up as a required check. If site builds are broken, PRs cannot be merged.

Signed-off-by: Matt Welke <matt.welke@spectrocloud.com>

TylerGillson commented on docs/2-validatorctl/1-download-and-install.md:
```diff
@@ -0,0 +1,20 @@
+
+Download validatorctl for your OS and architecture from GitHub [Releases](https://github.com/validator-labs/validatorctl/releases).
+
+## Install
```
curl command to download the latest release?

TylerGillson commented on docusaurus.config.ts:
```diff
@@ -13,7 +13,7 @@ const deployingToNetlify = process.env['DEPLOYING_TO_NETLIFY'] === 'TRUE' || fal
 const config: Config = {
   title: 'Validator Documentation',
-  tagline: 'Validate all the things',
+  tagline: 'Continuous validation for environments',
```
"Validate any environment" ?

"Ad hoc and continuous validation for any environment" ?

TylerGillson commented on src/components/HomepageFeatures/index.tsx:
```diff
@@ -10,32 +10,29 @@ type FeatureItem = {
 const FeatureList: FeatureItem[] = [
   {
-    title: 'Easy to Use',
+    title: 'Day 1, 2 validation',
```
Day 0-2 validation

TylerGillson commented on docs/1-architecture.md:
Please add 0-overview.md inspired by our gh organization page

TylerGillson reviewed (commented):
Please use orange/blue colour scheme and replace docusaurus svgs.

Commit:
Add download script, update install page

Signed-off-by: Matt Welke <matt.welke@spectrocloud.com>

Commit:
Updates.

Replace SVG graphics with new ones from undraw. Used orange as primary color when downloading the new graphics.

Replace colors in included them with blue equivalents, with help of gen AI.

Due to previous changes, using blue/orange color theme overall.

Refine language in splash page (correct CRD -> CR, use "Day 0-2" terminology, etc).

Add Overview page to docs pages, as first page. Includes content previously on GitHub org.

Add community page to link people to code of conduct, contributing instructions, GitHub Discussions, and the Spectro Cloud Slack workspace.

Signed-off-by: Matt Welke <matt.welke@spectrocloud.com>

Commit:
Correct link in install script.

Signed-off-by: Matt Welke <matt.welke@spectrocloud.com>

Commit:
Logos and style

Signed-off-by: Matt Welke <matt.welke@spectrocloud.com>

Commit:
Better install instructions

Signed-off-by: Matt Welke <matt.welke@spectrocloud.com>

Commit:
Notes about build availability.

Signed-off-by: Matt Welke <matt.welke@spectrocloud.com>

ahmad-ibra commented on docs/validatorctl/1-download-and-install.md:
````diff
@@ -0,0 +1,56 @@
+curl -L https://raw.githubusercontent.com/validator-labs/docs/main/scripts/get_validator.sh | bash
+```
+
+### Installing with `go install`
+
+To install, run:
+
+```shell
+go install github.com/validator-labs/validatorctl@latest
+```
+
+:::note
+
+The MacOS build is currently only available for arm64 and the Linux build is currently only available for amd64.
+
+:::
````
I dont think this should be only mac/linux specific. It would also apply to windows. Im pretty sure if you're on windows you can still run go install, and then based on your `GOOS` and `GOARCH` it'll setup the binary correctly (ie as a .exe) and put it in your PATH.

mattwelke replied:
Good point, ty. `go install` should work on Windows too.

ahmad-ibra reviewed (changes requested):
overall looks really good, just the one comment about the install instructions

Commit:
Re-arrange install instructions for `go install` on any OS

Signed-off-by: Matt Welke <matt.welke@spectrocloud.com>

ahmad-ibra reviewed (approved):
LGTM!
